client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/CRUDDocumentationIT.java

```diff
@@ -318,4 +325,87 @@ public void onFailure(Exception e) {
             // end::delete-conflict
         }
     }
+
+    public void testBulk() throws IOException {
+        RestHighLevelClient client = highLevelClient();
+        {
+            BulkRequest bulkRequest = new BulkRequest()
+                    .add(new IndexRequest("index", "type", "1").source(Requests.INDEX_CONTENT_TYPE, "field", "value"))
+                    .add(new IndexRequest("index", "type", "2").source(Requests.INDEX_CONTENT_TYPE,"field", "value"));
+            BulkResponse bulkResponse = client.bulk(bulkRequest);
+            assertFalse(bulkResponse.hasFailures());
```

Comment:
interesting approach, still finding my way here. You went for a snippet that only tests the  behaviour without any docs extraction right? Wondering if this is useful or not, cause we already have proper IT tests like this. It feels weird to be merely testing from one of these tests, but we should definitely make sure that what we document works the way we document it, which we can only do through "hidden" assertions, like you do below.